### ISSUE TITLE
RPM package requires PostgreSQL even if Zammad is configured for MySQL/MariaDB

### ISSUE THREAD

**Issue by fthommen** [zammad/zammad#3791]
### Infos:

* Used Zammad version: 4.1.0-1626855612.90b59692
* Installation method (source, package, ..): Package (RPM from Zammad YUM Repository)
* Operating system: CentOS 7.9.2009
* Database + version: MariaDB 10.3
* Elasticsearch version: 7.9.2
* Browser + version: Firefox 92.x

### Expected behavior:

`yum update zammad` runs through w/o problems as the required MariaDB version is installed and configured

### Actual behavior:

```
# yum update zammad
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
Resolving Dependencies
--> Running transaction check
---> Package zammad.x86_64 0:4.1.0-1626855612.90b59692.centos7 will be updated
---> Package zammad.x86_64 0:5.0.0-1633523150.497b6aae.centos7 will be an update
--> Processing Dependency: postgresql-server >= 9.3 for package: zammad-5.0.0-1633523150.497b6aae.centos7.x86_64
--> Finished Dependency Resolution
Error: Package: zammad-5.0.0-1633523150.497b6aae.centos7.x86_64 (zammad)
           Requires: postgresql-server >= 9.3
           Installed: postgresql-server-9.2.24-7.el7_9.x86_64 (@centos-updates-dkfz)
               postgresql-server = 9.2.24-7.el7_9
           Available: postgresql-server-9.2.24-4.el7_8.x86_64 (centos-base-dkfz)
               postgresql-server = 9.2.24-4.el7_8
           Available: postgresql-server-9.2.24-6.el7_9.x86_64 (centos-updates-dkfz)
               postgresql-server = 9.2.24-6.el7_9
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
# 
```

### Steps to reproduce the behavior:

`yum update zammad` on a server where MariaDB is used as Zammad database. For those installations PostgreSQL should not be a requirement.

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by MrGeneration** [comment]
I'm afraid we won't be able to fix that because if we remove the line the dependency is not enforced.
CentOS7 and its yum are not capable of "OR" clauses (even on CentOS 8 you need specific versions) to support that.

https://github.com/zammad/zammad/blob/develop/.pkgr.yml#L12

I'm afraid we'll have to mark this as wontfix, sorry!

**Comment by fthommen** [comment]
ok, fair point.  But I will then also assume, that a force-upgrade, skipping the PostgreSQL installation, should be safe when Zammad is using MariaDB? @MrGeneration?
